remoteappmanager/handlers/home_handler.py

```diff
@@ -244,13 +239,13 @@ def _start_container(self, orm_user, app, policy, mapping_id):
 
         Parameters
         ----------
-        orm_user : orm.User
-            the orm user object
+        orm_user : User
+            database's user object (e.g. current_user.orm_user)
```

Comment:
@itziakos 
@stefanoborini and I discussed offline whether we still need to pass the database's User object to `HomeHandler._start_container` or whther we can simply pass the username here (or not, username is accessible from HomeHandler.current_user).  We don't yet have a conclusion.

Note that there is no ABC defined for the database's User object (IMHO there is no need for one)